src/config.rkt

```diff
@@ -75,12 +75,20 @@
 ;; In localization, the maximum number of locations returned
 (define *localize-expressions-limit* (make-parameter 4))
 
+;; In localization, limit is for new locations
+;; (only if *use-improve-cache* is on)
+(define *localize-limit-for-new* (make-parameter #f))
+
 ;; How accurate to make the binary search
 (define *binary-search-test-points* (make-parameter 16))
 (define *binary-search-accuracy* (make-parameter 48))
 
-;; Enables Pherbie
+;; Pherbie related options
 (define *pareto-mode* (make-parameter #f))
+(define *pareto-pick-limit* (make-parameter 5))
+
+;; In mainloop, cache improvements between iterations
+(define *use-improve-cache* (make-parameter #t))
```

Comment:
I assume this is mainly for testing, right? In practice we would never want to configure this, and in fact plan to leverage this more.

Reply:
Yep, just to test if anything changes between runs. I should probably remove it.